bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifier.scala

```diff
@@ -238,4 +241,6 @@ trait BagVerifier[BagContext <: BagVerifyContext[BagPrefix], BagLocation <: Loca
           maybeUserFacingMessage = Some(userFacingMessage)
         )
     }
+
+  def addTrailingSlash(prefix: BagPrefix): BagPrefix
```

Comment:
One more minor nit: this doesn't always add a slash; sometimes it leaves it unmodified.

Maybe `ensureTrailingSlash`?

Reply:
not really, this gets called only after the check of whether the trailing slash already contains the slash

Reply:
Duh. Oh yeah, ignore me.